scripts/stac/imagery/collection.py

```diff
@@ -59,6 +68,47 @@ def __init__(
 
         self.add_providers(providers)
 
+    def add_capture_area(self, polygons: List[shapely.geometry.shape], target: str) -> None:
+        """Add the capture area of the Collection.
+        The `href` or path of the capture-area.geojson is always set as the relative `./capture-area.geojson`
+
+        Args:
+            polygons: list of geometries
+            target: path of the capture-area-geojson file
+        """
+
+        capture_area = geojson.Feature(geometry=shapely.ops.unary_union(polygons), properties={})
```

Comment:
if polygons are slightly miss aligned eg 15.999997 and 16.00001 does this actually join the polygons?

Reply:
I haven't test that. We should validate this with a unit test and maybe have a chat about it.

Reply:
as we are projecting things from 2193->4326 the numbers may vary slightly